base/docs/helpdb/Base.jl

```diff
@@ -2265,11 +2265,11 @@ sortperm
 """
     mod2pi(x)
 
-Modulus after division by 2pi, returning in the range \[0,2pi).
+Modulus after division by `2pi`, returning in the range `\[0,2pi)`.
```

Comment:
For small uncontroversial PRs, I would think it's fine. Else it will increase the cost of fixing small mistakes you find while reading the docs, and nobody will do it.